packages/cli/lib/lib/webpack/transform-config.js

```diff
@@ -21,12 +21,48 @@ module.exports = async function (env, config, ssr=false) {
 		}]]
 	});
 	const m = require(myConfig);
-	const transformer = m && m.default || m;
+	const config = m && m.default || m;
 	try {
 		let helpers = new WebpackConfigHelpers(env.cwd);
-		await transformer(config, Object.assign({}, env, { ssr }), helpers);
+		const transformers = [];
+		
+		if (typeof config === 'function') {
+			transformers.push(config);
+		} else if (typeof config === 'object' && !Array.isArray(config)) {
+			// TODO: do we want more props?
+			if (config.plugins) {
+				if (!Array.isArray(config.plugins)) {
+					throw new Error('The `plugins` property in `preact.config.js` has to be an Array');
+				}
+				
+				config.plugins.map((plugin, index) => {
+					if (typeof plugin === 'function') {
```

Comment:
as discussed in chat - I would *looove* to be able to plugin strings + options here and have them autorequire, since that'd make configuration completely static json:

```js
  plugins: [
    'typescript',
    ['a-plugin-with-options', {
      some: 'opts'
    }]
  ]
````